File: DSA/Gautam_Shyamnani_590013078/Day3Question1.cpp
#include <iostream>
#include <vector>

using namespace std;

int findSum(vector<int> nums, int k) {
    int sum = 0;
    int n = nums.size();

    for (int i = 0; i < n; i++) {
        int count = 0;
        bool isDuplicate = false;

        for (int j = 0; j < n; j++) {
            if (nums[i] == nums[j]) count++;
        }

        if (count == k) {
            for (int m = 0; m < i; m++) {
                if (nums[m] == nums[i]) isDuplicate = true;
            }
            if (!isDuplicate) sum += nums[i];
        }
    }
    
    return sum;
}

int main() {
    vector<int> nums = {2, 3, 9, 9};
    int k = 1;

    cout << findSum(nums, k) << endl;
    return 0;
}

File: DSA/Pranvkumar_590011587/Day3_Question1.py
from collections import Counter

def sum_of_Unique_nums_of_freq_k(nums,k):
    freq = Counter(nums)
    return sum(num for num, count in freq.items() if count == k)

# Example :
nums = [1, 2, 2, 4, 4]
k=2
result = sum_of_Unique_nums_of_freq_k(nums,k)
print(result)